Sources/DittoPresenceViewer/PresenceView.swift

```diff
@@ -19,18 +19,20 @@ import UIKit
 import AppKit
 #endif
 
-@available(macOS 10.15, *)
+#if canImport(WebKit)
+@available(iOS 13, macOS 10.15, *)
```

Comment:
I think we can drop the `iOS` piece here since we only support 14.0 and newer now